PropertySheets/Platform.props

```diff
@@ -61,8 +61,6 @@
       <StringPooling>true</StringPooling>
       <MinimalRebuild>true</MinimalRebuild>
       <ExceptionHandling>Async</ExceptionHandling>
-      <BasicRuntimeChecks>Default</BasicRuntimeChecks>

```

Comment:
It's actually not removed. Please see both Debug.props and Release.props.
It is defined on both files with different values.
Its presence on Platform.props was redundant and always overwritten.